pom.xml

```diff
@@ -110,6 +110,25 @@
             <classifier>mule-plugin</classifier>
             <scope>test</scope>
         </dependency>
+        <!-- Jackson dependency needs to be overwritten to avoid conflicts between ActiveMQ and Allure -->
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <version>2.9.2</version>
```

Comment:
which version's is this? ActiveMQ's or Allure's?

Reply:
Neither, both of them use a very old version of Jackson which is full of bugs and vulnerabilities.
Notice this is test scope. This is just to unify the Jackson version so the test do not fail because the Jackson API changed.
However, after this fix, another problem surfaces when upgrading to ActiveMQ 5.15 which is related to Spring transitive dependencies. That one is harder to fix.